### PR TITLE
FIX minor fixes in logs.md doc

### PR DESCRIPTION
**Pull request by fgalan**
@cesarjorgemartinez

### CODE REVIEW

**Comment by cesarjorgemartinez** [comment]
LGTM

File: doc/manuals/admin/logs.md [hunk 1]
```diff
@@ -60,8 +60,8 @@ separed by the pipe character (`|`). Example:
 The different fields in each line are as follows:
 
 -   **time**. A timestamp corresponding to the moment in which the log
-    line was generated.
--   **lvl (level)**. There are four level types:
+    line was generated in [ISO8601](https://es.wikipedia.org/wiki/ISO_8601) format.
+-   **lvl (level)**. There are five level types:
```
Comment (kzangeli) [inline doc/manuals/admin/logs.md]: level types ...
I would say  'There are five levels'

Comment (fgalan) [inline doc/manuals/admin/logs.md]: Fixed in 0ebaa7e4d03c3b106386ca9e438c35228b6fc2c0

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**